Fakes/Fakes/Networking.generated.swift

```diff
@@ -1161,6 +1161,29 @@ extension Networking.POSProduct {
         )
     }
 }
+extension Networking.POSProductVariation {
+    /// Returns a "ready to use" type filled with fake values.
+    ///
+    public static func fake() -> Networking.POSProductVariation {
+        .init(
+            siteID: .fake(),
+            productID: .fake(),
+            productVariationID: .fake(),
+            attributes: .fake(),
+            image: .fake(),
+            sku: .fake(),
+            globalUniqueID: .fake(),
+            price: .fake(),
+            regularPrice: .fake(),
+            salePrice: .fake(),
+            onSale: .fake(),
+            downloadable: .fake(),
+            manageStock: .fake(),
+            stockQuantity: .fake(),
+            stockStatusKey: .fake()
+        )
+    }
+}
```

Comment:
Looks like the changes from #15707 worked as expected 👍

Reply:
Yep! I did test them with a new addition before I merged your PR too 😊 

Thanks again